### DISCUSSION TITLE
Show: Add or update "created_at" via save hook

### DISCUSSION THREAD

**Discussion by minthemiddle** [notaapp/talk#428] [Random]
Here's a [gist](https://gist.github.com/minthemiddle/d24146a1ae85a65a78b733f6b5cdb4be) for a [save hook](https://docs.nota.md/save-hook) that adds or updates created_at on save in `YYYY-mm-dd` format.

**Comment by astoilkov** [comment]
Nice. I didn't even know that was possible to do with a save hook. Thanks for sharing this!
